### PR TITLE
Update SDK and enable fixed tests

### PR DESCRIPTION
**Pull request by JamesNK**
SDK contains HTTP/2 fix for some disabled tests.

### CODE REVIEW

File: test/FunctionalTests/Server/ClientStreamingMethodTests.cs [hunk 1]
```diff
@@ -275,8 +275,6 @@ async Task<CounterReply> AccumulateCount(IAsyncStreamReader<CounterRequest> requ
             var responseTask = Fixture.Client.SendAsync(httpRequest, HttpCompletionOption.ResponseHeadersRead);
 
             // Assert
-            Assert.IsFalse(responseTask.IsCompleted, "Server should wait for client to finish streaming");
```
Comment (JunTaoLuo) [inline test/FunctionalTests/Server/ClientStreamingMethodTests.cs]: Why is this removed?

Comment (JamesNK) [reply test/FunctionalTests/Server/ClientStreamingMethodTests.cs]: It's a bad assert. The client stream is being sent immediately.
